client/signup/jetpack-connect/controller.js

```diff
@@ -178,7 +176,7 @@ export default {
 	plansLanding( context ) {
 		const Plans = require( './plans' ),
 			CheckoutData = require( 'components/data/checkout' ),
-			site = sites.getSelectedSite(),
+			site = getSelectedSite( context.store.getState() ),
```

Comment:
It might be worthwhile to create another selector for `isSelectedSiteJetpack` in ui/selectors, but we can avoid fetching a full site object (with it's computed attributes) by using the following:
```javascript
import { getSelectedSiteId } from 'state/ui/selectors';
import { isJetpackSite } from 'state/sites/selectors';

const siteId = getSelectedSiteId( state );
const isJetpack = isJetpackSite( state, siteId );

if (  ! config.isEnabled( 'jetpack/connect' ) || ! isJetpack ) {
   return;
} 
```